api/controllers/stories.js

```diff
@@ -0,0 +1,30 @@
+const Story = require('../models/Story');
+const User = require('../models/User')
+
+
+module.exports.createStory =  async (req, res, next) => {
+    
+    if(!req.body.story) return res.status(400).send('No text provided');
```

Comment:
Requirement: can you make sure the error follow the pattern of the other errors ?
```suggestion
    if (!req.body.story) {
       return res.status(400).send({error: true, message: "No story submitted"});
    }
```